Makefile.validation

```diff
@@ -50,6 +50,19 @@ test-sanity:
 #	should fail when requesting to create a volume with already existing name and different capacity|\
 #	should not fail when requesting to create a volume with already existing name and same capacity" -ginkgo.v -timeout=0
 
+### Workflows
+
+prepare-env:
+	go install github.com/rhysd/actionlint/cmd/actionlint@latest
+	go install github.com/nektos/act@latest
+
+workflows-lint:
```

Comment:
Could we make this target a part of PR validation and run it for all workflows?

Reply:
Actually, this target will run for all workflows located in .github/workflows dir.

Reply:
Excellent! May I ask you to try to add this target to PR validation workflow?

Reply:
Do you mean separate target only for PR validation?